firewood/src/merkle/stream.rs

```diff
@@ -328,7 +328,7 @@ impl<'a, S: ShaleStore<Node> + Send + Sync, T> FusedStream for MerkleKeyValueStr
 }
 
 impl<'a, S, T> MerkleKeyValueStream<'a, S, T> {
-    pub(super) fn new(merkle: &'a Merkle<S, T>, merkle_root: DiskAddress) -> Self {
+    pub(crate) fn new(merkle: &'a Merkle<S, T>, merkle_root: DiskAddress) -> Self {
```

Comment:
Why did this change? This shouldn't be called outside of `Merkle`

Reply:
Great catch, was doing some debugging when I changed this.